atom/browser/native_window_mac.mm

```diff
@@ -309,6 +309,7 @@ @interface AtomNSWindow : EventDispatchingWindow<QLPreviewPanelDataSource, QLPre
 @property BOOL acceptsFirstMouse;
 @property BOOL disableAutoHideCursor;
 @property BOOL disableKeyOrMainWindow;
+@property BOOL hideAfterFullscreenExit;
```

Comment:
Is this variable used anywhere? I'm not seeing other references to it in the other diff hunks.